README.md

```diff
@@ -29,14 +29,39 @@ make install
 ```
 
 Install location can be controlled by passing `DESTDIR=<DIR>` before the
-`install` target. You can also control the location of the installed environment
-files on the final target with `ENV_DIR`. The default is `/boot/efi/EFI/BOOT`
-which assumes that the [EFI
+`install` target. By default, the installation is integrated with the
+`/etc/grub.d` scripts, which means you need to run `grub-install` and
+`update-grub` after installation on the platform. This normally has to be done
+while running the device, but you can use
+[mender-convert](https://github.com/mendersoftware/mender-convert) as a way to
+do this at build time.
+
+If you want to install only the tools, or only the script and environment files,
+run `make install-tools` or `make install-grub.d-boot-files`, respectively.
+
+### Standalone boot scripts
+
+It is also possible to install standalone boot scripts that do not integrate
+with the `/etc/grub.d` scripts. The advantage is that you do not need to run
+`grub-install` and `update-grub` on the platform afterwards, and the customized
+nature may be better suited for specialized situations, or where `/etc/grub.d`
+is not available.
+
+To install this variant, run this:
+
+```
+make
+make install-standalone
+```
+
+You can also control the location of the installed environment files on the
+final target with `ENV_DIR`. The default is `/boot/efi/EFI/BOOT` which assumes
+that the [EFI
```

Comment:
The main README should somehow reference the internal docs also (ones in `grub-mender-grubenv/grub.d/`).
This would allow for a smoother explanation on what those documents describe.

Reply:
Good thinking, I'll add a pointer here.